docs/UserGuide.md

```diff
@@ -41,18 +41,18 @@ Thus we recommend you to start by first understanding how McGymmy works at a hig
 
 ## Quick start
 
-Double click on the jar file to open the GUI.
-Type the command in the command box and press Enter to execute it. e.g. typing help and pressing Enter will open the help window.
+To get started, you will double click on the jar file to open the GUI.
```

Comment:
You will sound damn weird here
Maybe you can?

Reply:
Alright!